doc/api/repl.markdown

```diff
@@ -219,6 +219,9 @@ The special variable `_` (underscore) contains the result of the last expression
     > _ += 1
     4
 
+*NOTE*: The `_` should not be reassigned explicitly in REPL, as it will either
+mess up REPL's functionality or produce unexpected results.
```

Comment:
Perhaps simply: `*NOTE*: Explicitly assigning a value to `_` in the REPL can produce unexpected results.`